content/en/docs/tasks/extend-kubernetes/socks5-proxy-access-api.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Use a SOCKS5 Proxy to Access the Kubernetes API
+content_type: task
+weight: 40
+---
+
+<!-- overview -->
+This page shows how to use an SOCKS5 proxy to access a remote Kubernetes API.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Setup big-picture
+
+  {{< note >}}
+  In this example we're using SSH as the tunnel mechanism but it can be replaced by any other kind of SOCKS5 proxy.
+  {{</ note >}}
```

Comment:
Let's remove non-essential indents. 😊

```suggestion
{{< note >}}
In this example we're using SSH as the tunnel mechanism but it can be replaced by any other kind of SOCKS5 proxy.
{{</ note >}}
```

Reply:
done thanks!